packages/http-client-csharp/emitter/src/type/input-type.ts

```diff
@@ -130,26 +130,72 @@ export interface InputModelType extends InputTypeBase {
   additionalProperties?: InputType;
   discriminatorValue?: string;
   discriminatedSubtypes?: Record<string, InputModelType>;
-  discriminatorProperty?: InputModelProperty;
+  discriminatorProperty?: InputProperty;
   baseModel?: InputModelType;
   serializationOptions: SerializationOptions;
 }
 
-export interface InputModelProperty extends InputTypeBase {
-  kind: SdkModelPropertyType["kind"];
-  name: string;
-  serializedName: string;
+export interface InputModelPropertyTypeBase extends DecoratedType {
   type: InputType;
+  name: string;
+  doc?: string;
+  summary?: string;
+  // apiVersions: string[];
```

Comment:
nit: remove commented code

Reply:
I kind of want to keep them here because these are properties that TCGC types have but ours do not have yet.
In other words, these are "gaps".
It might be good that we keep them here as reminders.
What do you think?